iowa-a/bedrock-dev/pocket-deploy.yaml

```diff
@@ -0,0 +1,238 @@
+                  name: bedrock-pocket-dev-secrets
+            - name: CSP_DEFAULT_SRC
+              value: "*.allizom.org"
+            - name: CSP_REPORT_ENABLE
+              value: "True"
+            - name: DEBUG
+              value: "False"
+            - name: DEV
+              value: "True"
+            - name: DISABLE_SSL
+              value: "False"
+            - name: EMAIL_HOST
+              valueFrom:
+                secretKeyRef:
+                  key: email-host
+                  name: bedrock-pocket-dev-secrets
+            - name: EMAIL_HOST_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: email-host-password
+                  name: bedrock-pocket-dev-secrets
+            - name: EMAIL_HOST_USER
+              valueFrom:
+                secretKeyRef:
+                  key: email-host-user
+                  name: bedrock-pocket-dev-secrets
+            - name: EMAIL_PORT
+              valueFrom:
+                secretKeyRef:
+                  key: email-port
+                  name: bedrock-pocket-dev-secrets
+            - name: EMAIL_USE_TLS
+              valueFrom:
+                secretKeyRef:
+                  key: email-use-tls
+                  name: bedrock-pocket-dev-secrets
+            - name: ENABLE_CSP_MIDDLEWARE
+              value: "True"
+            - name: ENABLE_HOSTNAME_MIDDLEWARE
+              value: "True"
+            - name: FXA_ENDPOINT
+              value: "https://accounts.stage.mozaws.net/"
+            - name: GTM_CONTAINER_ID
+              value: GTM-MW3R8V
+            - name: HTTPS
+              value: "on"
+            - name: L10N_CRON
+              value: "True"
+            - name: LOGLEVEL
+              value: info
+            - name: MOFO_SECURITY_ADVISORIES_BRANCH
+              value: master
+            - name: MOFO_SECURITY_ADVISORIES_PATH
+              value: /tmp/mofo_security_advisories
+            - name: NEW_RELIC_APP_NAME
+              value: "bedrock-pocket-dev-iowa-a;bedrock-pocket-dev"
+            - name: NEW_RELIC_LICENSE_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: new-relic-license-key
+                  name: bedrock-pocket-dev-secrets
+            - name: PROD
+              value: "True"
+            - name: SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: secret-key
+                  name: bedrock-pocket-dev-secrets
+            - name: SECURE_BROWSER_XSS_FILTER
+              value: "True"
+            - name: SECURE_HSTS_SECONDS
+              value: "31536000"
+            - name: SECURE_SSL_REDIRECT
+              value: "True"
+            - name: SENTRY_DSN
+              value: "https://c1f94ae1b8c54ed0adcf223328157caf@o1069899.ingest.sentry.io/6260211"
+            - name: SITE_MODE
+              value: "Pocket"
+            - name: STATIC_URL
+              value: "https://www-dev.allizom.org/media/"
```

Comment:
@pmac It was on my list to explore whether (in prod, at least) we wanted to make `STATIC_URL` a pocket-specific CNAME for the same bucket, for those avid readers of the HTML source, but there's not really a huge benefit otherwise. Agreed?